custom_metrics_extractor/README.md

```diff
@@ -0,0 +1,29 @@
+# Custom metrics extractor
+This script generates a file containing all the custom metrics for an org. 
+
+It is related to [an older script to generate custom metrics](https://github.com/DataDog/se-docs/wiki/Pulling-the-list-of-custom-metrics----SCRIPT) with the added functionality to filter out core integration metrics and provides a count of total custom metrics.
+
+## Prerequisites
+In order to use this script, you would need a local copy of the [Datadog/dogweb repo](https://github.com/DataDog/dogweb) and [Datadog/integrations-core](https://github.com/DataDog/integrations-core) repo.
```

Comment:
Ok now I read a little further... customers won't have access to `dogweb` -- I think this PR and code should move to https://github.com/DataDog/Miscellany-Internal -- this repo is public and customers have access to it.